platforms/js/build_js.py

```diff
@@ -194,11 +196,25 @@ def build_doc(self):
     parser.add_argument('--skip_config', action="store_true", help="Skip cmake config")
     parser.add_argument('--config_only', action="store_true", help="Only do cmake config")
     parser.add_argument('--enable_exception', action="store_true", help="Enable exception handling")
-    args = parser.parse_args()
+    parser.add_argument('--cmake_option', action='append', help="Adds the ability to modify the list of modules")
+    parser.add_argument('--opencv_js_whitelist', help="Add an ability to connect a JSON file that include list of the OpenCV functions needed for developing a project") 
```

Comment:
Maybe just `--whitelist`?

Reply:
Or `--whitelistpath`?

Reply:
No, just `--whitelist` enough. Specify in description that this should be a path to a .json file with specific format.